server/controllers/userController.js

```diff
@@ -1,53 +1,66 @@
     process.env.SECRET_KEY,
     {expiresIn: '24h'}
 )
 
 class UserController {
     async registration(req, res, next) {
-        const {email, password, role} = req.body
-        if (!email || !password) {
-            return next(ApiError.badRequest('Некорректный email или password!'))
-        }
+        try {
+            const {email, password, role} = req.body
+            if (!email || !password) {
+                return next(ApiError.badRequest('Некорректный email или password!'))
+            }
 
-        const candidate = await User.findOne({where: {email}})
-        if (candidate) {
-            return next(ApiError.badRequest('Пользователь с таким email уже существует!'))
-        }
+            const candidate = await User.findOne({where: {email}})
+            if (candidate) {
+                return next(ApiError.badRequest('Пользователь с таким email уже существует!'))
+            }
 
-        const hashPassword = await  bcrypt.hash(password, 5)
-        const user = await User.create({email, role, password: hashPassword})
-        const order = await Orders.create({userId: user.id})
-        const token = generateJws(user.id, user.email, user.role)
+            const hashPassword = await  bcrypt.hash(password, 5)
+            const user = await User.create({email, role, password: hashPassword})
+            await Orders.create({userId: user.id})
+            const token = generateJwt(user.id, user.email, user.role)
 
-        return res.json({token})
+            return res.json({token})
+        } catch (e) {
+            next(ApiError.badRequest((e.message)))
+        }
     }
 
     async login(req, res, next) {
-        const {email, password} = req.body
-        const user = await User.findOne({where: {email}})
-        if (!user) {
-            return next(ApiError.internal('Пользователь не найден!'))
-        }
+        try {
+            const {email, password} = req.body
+            const user = await User.findOne({ where: { email } })
+            if (!user) {
+                return next(ApiError.notFound('Пользователь не найден!'))
+            }
 
-        let comparePassword = bcrypt.compareSync(password, user.password)
-        if (!comparePassword) {
-            return next(ApiError.internal('Указан неверный пароль!'))
-        }
+            const isPasswordValid = bcrypt.compareSync(password, user.password)
+            if (!isPasswordValid) {
+                return next(ApiError.internal('Указан неверный пароль!'))
+            }
 
-        const token = generateJws(user.id, user.email, user.role)
-        return res.json({token})
+            const token = generateJwt(user.id, user.email, user.role)
+            return res.json({token})
+        } catch (e) {
+            next(ApiError.badRequest((e.message)))
+        }
     }
 
-    async check(req, res) {
-        const token = generateJws(req.user.id, req.user.email, req.user.role)
-        res.json({token})
+    async checkAuth(req, res, next) {
```

Comment:
the name of the endpoint doesn't resemble what it does.
It probably should be something like: 
method: async regenerateToken
endpoint: /refresh-token